### PR TITLE
Update to use sklearn pipeline

### PR DESCRIPTION
**Pull request by lucyleeow**
ping @agramfort @glemaitre

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+dependencies:
+  - pip
+  - numpy
+  - scikit-learn
+  - pandas
+  - nltk
+  - jupyter
+  - matplotlib
+  - pip:
+    - ramp-workflow
```
Comment (agramfort) [inline environment.yml]: empty line missing

**Comment by agramfort** [comment]
some of the plots are not readable in https://github.com/ramp-kits/fake_news/blob/ea2e535a21853e096192bfa9636a0b60068c7d0b/fake_news_starting_kit.ipynb can you see if you can maybe plot the top figures and not the full ? to make sure xlabels are always readable

**Comment by agramfort** [comment]
also for the end fo the notebook you can use void the Local testing section and point to ramp doc like for other recent notebooks

**Comment by agramfort** [comment]
regarding unicode we are now in python3 so it should be simpler now

**Comment by agramfort** [comment]
besides LGTM @lucyleeow !

**Comment by lucyleeow** [comment]
thanks @agramfort merging

**Comment by lucyleeow** [comment]
merging thanks @agramfort

**Pull request merged**